libs/ledger-live-common/src/families/ethereum/specs.ts

```diff
@@ -95,66 +85,69 @@ const ethereumBasicMutations = ({ maxAccount }) => [
   },
 ];
 
-function findCompoundAccount(account, f: (...args: Array<any>) => any) {
-  return sample(
-    (account.subAccounts || []).filter((a) => {
-      if (
-        a.type === "TokenAccount" &&
-        a.balance.gt(0) &&
-        findCompoundToken(a.token)
-      ) {
-        const c = getAccountCapabilities(a);
-        return c && f(c, a);
-      }
-
-      return false;
-    })
-  );
-}
-
-function getCompoundResult({ account, transaction, accountBeforeTransaction }) {
-  const a = account.subAccounts?.find((a) => a.id === transaction.subAccountId);
-  const aBefore = accountBeforeTransaction.subAccounts?.find(
-    (a) => a.id === transaction.subAccountId
-  );
-  invariant(
-    a && a.type === "TokenAccount",
-    "account %s found",
-    transaction.subAccountId
-  );
-  invariant(
-    aBefore && aBefore.type === "TokenAccount",
-    "account %s found",
-    transaction.subAccountId
-  );
-  const capabilities = getAccountCapabilities(a);
-  const summary = makeCompoundSummaryForAccount(a, account);
-  const capabilitiesBefore = getAccountCapabilities(aBefore);
-  const summaryBefore = makeCompoundSummaryForAccount(
-    aBefore,
-    accountBeforeTransaction
-  );
-  invariant(
-    capabilities,
-    "account %s have no capabilities found",
-    transaction.subAccountId
-  );
-  return {
-    a,
-    capabilities,
-    summary,
-    previous: {
-      summary: summaryBefore,
-      capabilities: capabilitiesBefore,
-    },
-  };
-}
-
 const minAmountETH = parseCurrencyUnit(
   getCryptoCurrencyById("ethereum").units[0],
   "0.01"
 );
 
+const erc20mutation: MutationSpec<Transaction> = {
```

Comment:
You could also use this in place of the existing erc20 mutations I guess?

Reply:
not sure i understand; there were lot of typing issue that i fixed there.